src/main/java/jackbean/task/TaskList.java

```diff
@@ -0,0 +1,92 @@
+package jackbean.task;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a list of tasks in the JackBean chatbot.
+ * This JavaDoc was written by GitHub Copilot.
+ */
+public class TaskList {
+    private ArrayList<Task> taskList = new ArrayList<Task>();
+
+    /**
+     * Constructs a task list.
+     * This JavaDoc was written by GitHub Copilot.
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<Task>();
+    }
+
+    /**
+     * Returns the number of tasks in the task list.
+     * This JavaDoc was written by GitHub Copilot.
+     *
+     * @return The number of tasks in the task list.
+     */
+    public String howManyTasks() {
```

Comment:
Should the name of this method be changed to include an action/verb such as getNumberOfTasks?